src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/DoNotUseEnumerableMethodsOnIndexableCollectionsInsteadUseTheCollectionDirectlyTests.Fixer.cs

```diff
@@ -13,8 +14,8 @@ namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
 {
     public class DoNotUseEnumerableMethodsOnIndexableCollectionsInsteadUseTheCollectionDirectlyFixerTests
     {
-        [Fact]
-        public async Task CA1826FixEnumerableFirstExtensionCallCSharp()
+        [Fact, WorkItem(1932, "https://github.com/dotnet/roslyn-analyzers/issues/1932")]
+        public async Task CA1826_CSharp_EnumerableFirstExtensionCall()
```

Comment:
I have renamed all tests to follow the same pattern and I have added a work item attribute to link to the ticket.